### DISCUSSION TITLE
Can Image Magick have webP support on cpanel centos 6

### DISCUSSION THREAD

**Discussion by oliflorence2601** [ImageMagick/ImageMagick#2568] [Help]
Hello,
I have a function (using PHP GD) that creates webP images from a jpg on wamp and this works fine. I have been told by my host that GD cannot get webP support unless the server runs centos 7 (we are using cPanel).
I read that PHP image magick can also be used to create webP images but I also found posts with people getting issues with webP support on centos 6 with Image Magick.
Could someone please let me know if it is possible to have webP support on cPanel Centos 6 if we add Image Magick?

**Comment by urban-warrior** [comment]
Install the Webp delegate library and headers on your CentOS 6 system.  Next, install ImageMagick from source.  It should discover you have WebP installed and build WebP support into ImageMagick.

**Reply by Julian-13** [reply]
can you explain in more detail please:
- how it discovering WebP?

i have some troubles with this case
(cpanel 86, centos 6)

trying to install:
/opt/cpanel/ea-php70/root/usr/bin/pecl install imagick
or
/opt/cpanel/ea-php70/root/usr/bin/pecl install --configureoptions 'with-webp="yes"' imagick

check it with "/opt/cpanel/ea-php70/root/usr/bin/php -i"
but no luck:
- ImageMagick supported formats: no webp support string
